src/plugins/internalnotification/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+include (LibAddMacros)
+
+IF (NOT PLUGIN_INTERNALNOTIFICATION_VERBOSE)
+	SET (PLUGIN_INTERNALNOTIFICATION_VERBOSE "0" CACHE STRING "Enable verbose output for internalnotification plugin")
+ENDIF (NOT PLUGIN_INTERNALNOTIFICATION_VERBOSE)
+
+MARK_AS_ADVANCED(
+	PLUGIN_INTERNALNOTIFICATION_VERBOSE
```

Comment:
try to avoid the variable, simple use VERBOSE set in kdbconfig.h.
Improvements of Elektra's "logger" are very welcomed.